dotm_search.py

```diff
@@ -34,14 +34,18 @@ def examine_files(dir, char):
     return (f_srched + "\n" + f_fnd + "\n" + f)
 
 
-def main():
-    parser.add_argument('--dir', help='directory to find data',
+def parsing():
+    parser.add_argument('--dir', help='directory of files to search',
                         type=str, default='./dotm_files')
     parser.add_argument('--char',
```

Comment:
i like how you've added help information to these arguments and forced them to be strings.  you may want to consider making your "--char" argument positional rather than optional, just to be more in line with the README's directions.